storagenode/pieces/deleter.go

```diff
@@ -43,14 +43,6 @@ type Deleter struct {
 
 // NewDeleter creates a new Deleter.
 func NewDeleter(log *zap.Logger, store *Store, numWorkers int, queueSize int) *Deleter {
-	if numWorkers == 0 {
```

Comment:
You should add a check against <=0 here.

Reply:
Why don't we validate all config options? Why should a user add a value <= 0? I don't think this is needed at all.

Reply:
I think it helps to detect mis-configuration (e.g. forgetting to update testplanet config)